modules/spark-wallet.nix

```diff
@@ -31,24 +30,30 @@ in {
         If enabled, the spark-wallet service will be installed.
       '';
     };
-    host = mkOption {
+    address = mkOption {
       type = types.str;
       default = "localhost";
-      description = "http(s) server listen address.";
+      description = "http(s) server address.";
     };
-    onion-service = mkOption {
-      type = types.bool;
-      default = false;
-      description = ''
-        "If enabled, configures spark-wallet to be reachable through an onion service.";
-      '';
+    port = mkOption {
+      type = types.port;
+      default = 9737;
+      description = "http(s) server port.";
     };
     extraArgs = mkOption {
       type = types.separatedString " ";
       default = "";
       description = "Extra command line arguments passed to spark-wallet.";
     };
-    enforceTor =  nix-bitcoin-services.enforceTor;
+    getPublicAddressCmd = mkOption {
+      type = types.str;
+      default = "";
+      description = ''
+        Bash expression which outputs the public service address to display in the web interface.
```

Comment:
µ-nit

Reply:
[The](https://twitter.com/nixbitcoinorg/status/1346851008446668802) 27 year old nix-bitcoin developer: enthusiastically addressing µ-nits at 1 am.
Actually, Jonas' comment was very relevant — because my description was totally wrong. The QR code is never shown in the web interface. Fixed in 3b5bb802407b4188abbf3998b28aa70750e6a4a9.